developerkhm/AndroidStudy/app/src/main/java/com/example/androidstudy/model/data/NaverDataImpl.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.androidstudy.model.data
+
+import com.example.androidstudy.api.RetrofitBuilder
+import com.example.androidstudy.api.data.TotalModel
+import retrofit2.Call
+
+object NaverDataImpl : INaverData {
+    override fun callAPiNaverSearch(type: String, query: String): Call<TotalModel> {
```

Comment:
callApiNaverSearch가 좋을것 같아요!
```suggestion
    override fun callApiNaverSearch(type: String, query: String): Call<TotalModel> {
```